docs/about/contributors.md

```diff
@@ -0,0 +1,10 @@
+---
+id: contributors
+custom_edit_url: null
+---
+# Contributors
+
+These people have contributed to Open Source with SLU projects. They have been added in (approximately) the order in which they became contributors.
+
+- [Kate Holdener, Ph.D](https://github.com/kate-holdener)
+- [Daniel Shown](https://github.com/kungfuchicken)
```

Comment:
It would be good to link the contributors page from the Home page, to recognize contributors.

Reply:
Also, it would be good to include our grad students that have graduated, wouldn't it? What about undergrads?

Reply:
yes, and yes. for alumni, i think we should verify they want to be listed. i can tackle that as a separate task.